extensions/sql-database-projects/src/models/project.ts

```diff
@@ -1029,6 +1063,33 @@ export class Project implements ISqlProject {
 		return false;
 	}
 
+	/**
+	 * Delete a Remove node from the sqlproj, ex: <Build Remove="Table1.sql" />
+	 * @param xmlTag The XML tag of the node (Build, None, PreDeploy, PostDeploy)
+	 * @param relativePath The relative path of the previously excluded file
+	 */
+	private async undoExcludeFileFromProjFile(xmlTag: string, relativePath: string): Promise<void> {
+		const nodes = this.projFileXmlDoc!.documentElement.getElementsByTagName(xmlTag);
+		for (let i = 0; i < nodes.length; i++) {
+			const parent = nodes[i].parentNode;
+			if (parent && nodes[i].getAttribute(constants.Remove) === utils.convertSlashesForSqlProj(relativePath)) {
+				parent.removeChild(nodes[i]);
+
+				// delete ItemGroup if this was the only entry
+				// only want element nodes, not text nodes
+				const otherChildren = Array.from(parent.childNodes).filter((c: any) => c.childNodes);
```

Comment:
childNodes should have type `ChildNode`, you should use that instead of `any`

Reply:
I sent a PR to update the rest of this file where any was being used #18091